xml_handlers/testing_handler.php

```diff
@@ -132,7 +132,7 @@ public function startElement($parser, $name, $attributes)
             $this->Label = new Label();
         } elseif ($name == 'TESTLIST' && $parent == 'TESTING') {
             $start_time = gmdate(FMT_DATETIME, $this->StartTimeStamp);
-            $end_time = gmdate(FMT_DATETIME, $this->EndTimeStamp); // The EndTimeStamp
+            // $end_time = gmdate(FMT_DATETIME, $this->EndTimeStamp); // The EndTimeStamp
```

Comment:
Instead of commenting this out, it looks like ``$end_time`` should be used [here](https://github.com/Kitware/CDash/pull/299/commits/0735a428c7256709bd697c896eefb7e6056b2122#diff-4d2eaf256ebf45e984948e246bfd0a3aR139).